### PR TITLE
chore: create contributing.md

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Create `contributing.md` (copy/paste from Renovate bot repo)
- Strip out parts of `contributing.md` that are not relevant to this repo.
- Create new section in `README.md` that points to this guide

## Context:

I think it's nice to explain how to contribute to the repo.

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Instead, use the re-request review functionality.
+Read more about this in the [GitHub docs, Re-requesting a review](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#re-requesting-a-review).
+
+## Slack collaboration with maintainers
+
+Sometimes the codebase can be a challenge to navigate, especially for a first-time contributor.
+We don't want you spending an hour trying to work out something that would take us only a minute to explain.
+
+For that reason, we have a Slack channel dedicated to helping anyone who's working on or considering Pull Requests for Renovate.
+Please email rhys@arkins.net and simply mention that you need an invitation to the channel and you'll be added ASAP.
```
Comment (HonkingGoose) [inline .github/contributing.md:38]: Does the "get help on Slack" apply to this repo as well?

File: .github/contributing.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Contributing
+
+## Security
+
+If you think you've found a **security issue**, please do not mention it in this repository.
+Instead, email renovate-disclosure@whitesourcesoftware.com with as much details as possible so that it can be handled confidentially.
```
Comment (HonkingGoose) [inline .github/contributing.md:6]:
```suggestion
```

Question: do we want to redirect security bugs in the documentation website to email as well?

Comment (rarkins) [reply .github/contributing.md:6]: Yes, just in case

File: .github/contributing.md (hunk 1 above)
Comment (rarkins) [reply .github/contributing.md:38]: Sure

**Pull request merged**